### DISCUSSION TITLE
association usager/user no work if user not imported before

### DISCUSSION THREAD

**Discussion by hcimadomo-procsi** [glpi-project/glpi-agent#320] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hello,
I change my agent from ocs to agent glpi. In ocs, when a new computer is inventoried, the user is automaticly associed with the user find in ldap source. And the user is automaticly imported
In Glpi Agent, if the user is not imported before the inventory, the user field keep empty. 
Perhaps i miss a parameter ?
Best regards

**Comment by g-bougard** [comment]
Hi @hcimadomo-procsi 
I don't think this is a problem related to GLPI Agent. You're comparing OCS to GLPI, so this should be a server problem.

**Reply by hcimadomo-procsi** [reply]
Hello, i not use OCS, it's only a reference of a good process. I think too it's a server configuration but is it a issue or a misconfiguration ?
That is the question ...

**Reply by g-bougard** [reply]
I far I know a lot of people uses this feature with GLPI. So that may be a misconfiguration.
You may want to also ask the question in [GLPI Forum](https://forum.glpi-project.org/).
